mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryModSeqProvider.java

```diff
@@ -67,7 +67,6 @@ private AtomicLong getHighest(InMemoryId id) {
     }
 
     private long nextModSeq(InMemoryId mailboxId) {
-        new Exception().printStackTrace();
```

Comment:
SHould be a fixup of the commit where it was introduced

Reply:
done